component/azstorage/block_blob.go

```diff
@@ -533,13 +533,33 @@ func (bb *BlockBlob) List(prefix string, marker *string, count int32) ([]*intern
 	return blobList, listBlob.NextMarker.Val, nil
 }
 
+// track the progress of downloads of blobs
+func trackDownload(name string, bytesTransferred int64, count int64, downloadPtr *int64) {
+	if bytesTransferred >= (*downloadPtr)*100*1024*1024 || bytesTransferred == count {
```

Comment:
nit: use the MB constant here

Reply:
what is this bytesTransferred check for? Can we add a comment about that?

Reply:
This check is basically for tracking every 100 MB of data transferred, in this case downloaded.
The condition `bytesTransferred == count` checks if the download has completed or not. So, we are also tracking the download completion here. I will add some comments for this.